src/main/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -100,7 +110,7 @@ export class ClaimantResponseConverter {
       }
       return paymentIntention
     } else {
-      if (decisionType === DecisionType.CLAIMANT) throw new Error('claimant payment intention not found where decision type is CLAIMANT')
+      if (decisionType === DecisionType.CLAIMANT || decisionType === DecisionType.CLAIMANT_IN_FAVOUR_OF_DEFENDANT) throw new Error('claimant payment intention not found where decision type is CLAIMANT')
```

Comment:
Error message will never reflect the `DecisionType.CLAIMANT_IN_FAVOUR_OF_DEFENDANT`, please update